mw_jwt.go

```diff
@@ -206,22 +225,41 @@ func (k *JWTMiddleware) getBasePolicyID(token *jwt.Token) (string, bool) {
 	return "", false
 }
 
+func (k *JWTMiddleware) getUserIdFromClaim(token *jwt.Token) (string, error) {
+	var userId string
+	var found bool
+
+	if k.Spec.JWTIdentityBaseField != "" {
+		userId, found = token.Claims.(jwt.MapClaims)[k.Spec.JWTIdentityBaseField].(string)
+
+		if !found {
+			log.Warning("Base Field not found, falling back to SUB")
+		}
+	}
+	if !found {
+		userId, found = token.Claims.(jwt.MapClaims)[SUB].(string)
+		if !found {
+			message := fmt.Sprint("user id was not found in claims")
```

Comment:
no need to `fmt.Sprint` - just set the msg to a string

Reply:
Didnt want to repeat the text (then if you need to change it you do it once. maybe I should have added const.
**const message := fmt.Sprint("user id was not found in claims")**

Reply:
What I meant was, what is wrong with: `message := "user id was not found in claims"`

Reply:
yep! changed it to that. Before I had some variable..